sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/config/ThroughputGlobalControlGroup.java

```diff
@@ -0,0 +1,47 @@
+package com.azure.cosmos.implementation.throughputControl.config;
+
+import com.azure.cosmos.CosmosAsyncContainer;
+
+import java.time.Duration;
+
+import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkNotNull;
+
+public class ThroughputGlobalControlGroup extends ThroughputControlGroupInternal {
+    private static final Duration DEFAULT_CONTROL_ITEM_RENEW_INTERVAL = Duration.ofSeconds(10);
+    private static final Duration DEFAULT_CONTROL_ITEM_EXPIRE_INTERVAL = Duration.ofSeconds(60);
+
+    private final CosmosAsyncContainer globalControlContainer;
+    private final Duration controlItemRenewInterval;
+    private final Duration controlItemExpireInterval;
+
+    public ThroughputGlobalControlGroup(
+        String groupName,
+        CosmosAsyncContainer targetContainer,
+        Integer targetThroughput,
+        Double targetThroughputThreshold,
+        boolean isDefault,
+        CosmosAsyncContainer globalControlContainer,
+        Duration controlItemRenewInterval,
+        Duration controlItemExpireInterval) {
+
+        super (groupName, targetContainer, targetThroughput, targetThroughputThreshold, isDefault);
+
+        checkNotNull(globalControlContainer, "Global control container can not be null");
+
+        this.globalControlContainer = globalControlContainer;
+        this.controlItemRenewInterval = controlItemRenewInterval != null ? controlItemRenewInterval : DEFAULT_CONTROL_ITEM_RENEW_INTERVAL;
+        this.controlItemExpireInterval = controlItemExpireInterval != null ? controlItemExpireInterval : DEFAULT_CONTROL_ITEM_EXPIRE_INTERVAL;
```

Comment:
will change to: 
this.controlItemExpireInterval =
            controlItemExpireInterval != null ? controlItemExpireInterval : Duration.ofSeconds(2 * this.controlItemRenewInterval.toSeconds());